packages/frames.js/src/next/getCurrentUrl.ts

```diff
@@ -6,7 +6,7 @@ export function getCurrentUrl(
   req: Request | NextRequest | NextApiRequest | IncomingMessage
 ): URL | undefined {
   const scheme = process.env.NODE_ENV === "production" ? "https://" : "http://";
-  const appUrl = process.env.VERCEL_URL || process.env.APP_URL;
+  const appUrl = process.env.APP_URL || process.env.VERCEL_URL;
```

Comment:
I'm wondering whether it would be fine to change the `FrameOptions` type in `createFrames()` exported from `next` so you could pass the URL like this:

```
createFrames({
   // override the URL, either URL object or string, must be fully qualified URL with protocol
   url: new URL('https://my-domain.com')
})
```

Reply:
I mean that could be also used in different handlers as a fallback if you have to override the domain completely. What do you think @stephancill ?

Reply:
yes that would be first prize. i think we should implement that in a separate pr and fall back to this method if it's not defined